src/main/java/core/basesyntax/service/WriterService.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface WriterService {
+    void writeToFile(String fileName, String report);
```

Comment:
https://en.wikipedia.org/wiki/Dependency_inversion_principle
`report` name
abstraction (`WriterService`) depends on implementation (`WriterServiceImpl`). Your implementation told its abstraction that the second parameter would be some `report`. But what if you have some other implementation that would want to write an `essay` for example? So this Writer should be abstracted from its implementation; it shouldn't choose to know what they will want to write to the file. You should choose a more generic name, for example, `content`. It will mean that everyone who wants implement this interface will have to implement `write` method, which writes some generic `content` to some file